cmd/nerdctl/image/image_list_test.go

```diff
@@ -95,12 +95,12 @@ LABEL version=0.1`, testutil.CommonImage)
 	base.Cmd("build", "-t", tempName, "-f", buildCtx+"/Dockerfile", buildCtx).AssertOK()
 	defer base.Cmd("rmi", tempName).AssertOK()
 
-	busyboxGlibc, busyboxUclibc := "busybox:glibc", "busybox:uclibc"
-	base.Cmd("pull", busyboxGlibc).AssertOK()
-	defer base.Cmd("rmi", busyboxGlibc).AssertOK()
-
-	base.Cmd("pull", busyboxUclibc).AssertOK()
-	defer base.Cmd("rmi", busyboxUclibc).AssertOK()
+	// This test is about testing local filtering of image names - as such, we do not need remote images at all
+	taggedGlibc, taggedUclibc := "taggedimage:glibc", "taggedimage:uclibc"
+	base.Cmd("tag", testutil.CommonImage, taggedGlibc).Run()
+	base.Cmd("tag", testutil.CommonImage, taggedUclibc).Run()
+	defer base.Cmd("rmi", taggedGlibc).AssertOK()
+	defer base.Cmd("rmi", taggedUclibc).AssertOK()
```

Comment:
Why not just use simple meta syntax vars such as "foo" "bar"?
The implementation of libc doesn't really relate to the test.

Reply:
The existing test was using that - I just did not change it - and I don't think it matters one way or the other.
And yes, it is pretty much `vfoov` and `zzfoozz`

I am also not particularly interested in spending more energy than needed on these tests in their current form. They are neither particularly good nor complete enough, and I would rather focus on the rewrite.

Anyhow, I will update if you prefer.

Reply:
Done

Reply:
> The existing test was using that - I just did not change it - and I don't think it matters one way or the other. And yes, it is pretty much `vfoov` and `zzfoozz`
> 
> I am also not particularly interested in spending more energy than needed on these tests in their current form. They are neither particularly good nor complete enough, and I would rather focus on the rewrite.
> 
> Anyhow, I will update if you prefer.

Hope this didn't sound defensive.

I guess lately I have been working on tests / QA way too much for my own good :-).

Cheers.